packages/client-core/src/hooks/useRender3DPanelSystem.tsx

```diff
@@ -78,6 +80,8 @@ export function useRender3DPanelSystem(canvas: React.MutableRefObject<HTMLCanvas
     return () => {
       // cleanup entities and state associated with this 3d panel
       removeEntity(cameraEntity)
+      /**@todo why is this necessary to prevent gltf system stopping? */
```

Comment:
this is probably fixed now since we are using a suspense boundary and delayed entity recycling